packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -279,3 +281,13 @@ export const CallControls = (props: CallControlsProps & ContainerRectProps): JSX
 };
 
 const isEnabled = (option: unknown): boolean => option !== false;
+
+/** @private */
```

Comment:
This function seems appear multiple time in composite (correct me if I am wrong), can we put it somewhere as a util function?

Reply:
Earlier it was supposed to be used for config screen text and hold button. But since we are not doing that change right now, it is only being used here. But we can definitely move this to util function if we think it can be reused